stable/oidc-starter/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Default values for oidc-starter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: anchorfree/oidc-starter
+  tag: 0.0.2
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+oidc_starter:
+  debug: true
+  issuer: http://127.0.0.1:5556
+  clientId: example-client-id
+  clientSecret: ZXhhbXBsZS1hcHAtc2VjcmV0
+  redirectURI: http://127.0.0.1:5555/callback
+  baseRedirectURI: /callback
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  paths: []
+  hosts:
+  - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  limits:
+    cpu: 16m
```

Comment:
are you sure that 16 millicors is enough? 
reference:
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

Reply:
the same as above